### DISCUSSION TITLE
Playback position not saved when pausing in a group

### DISCUSSION THREAD

**Discussion by JoeMBlair** [music-assistant/support#2774] [Q&A] [answered]
Whenever I pause a track when playing in a universal group player it sets it back to the beginning of the track or whevever it was last manually scrubbed to. I'm running the latest Music Assistant Beta. Play and pause work perfectly when just playing to the player provider.

**Comment by marcelveldt** [comment]
yes, its a known issue that is under investigation. Hopefully fixed soon

**Reply by OzGav** [reply, marked as answer]
@JoeMBlair beta 30 has completely reworked groups logic. Let us know if this is resolved.

**Reply by JoeMBlair** [reply]
Yea it seems to be fixed, thank you!
